src/parser/jobs/gnb/modules/NoMercy.tsx

```diff
@@ -119,7 +119,10 @@ export default class NoMercy extends Module {
 			switch (actionId) {
 				// Blasting Zone is just a traited version of Danger Zone
 				// Impossible to see both in the same log outside of hacking
+				// Ryan Note: Because of this, I'm going to roll the logic of danger zone under blasting zone so lvl 70's still get their zone counted for.
 				case ACTIONS.BLASTING_ZONE.id:
+					lastNoMercy.numBlastingZones++
+					break
```

Comment:
```suggestion
```

This addition doesn't actually do anything - in JS, case statements fall through unless you break them, so

```js
// this switch
switch (colour) {
	case RED:
	case BLUE:
		paint(colour)
		break
}
// is doing the same thing as this one
switch (colour) {
	case RED:
		paint(colour)
		break
	case BLUE:
		paint(colour)
		break
}
```

Reply:
TIL. I assumed since the notes left said it wasn't a thing that that code did nothing. Reverted.